Vagrantfile

```diff
@@ -0,0 +1,41 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+arteriauser    = ENV['ST2USER'] ? ENV['ST2USER']: 'arteriaadmin'
+arteriapasswd  = ENV['ST2PASSWORD'] ? ENV['ST2PASSWORD'] : 'arteriarulz'
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.define "arteria" do |arteria|
+    # Box details
+    arteria.vm.box = "bento/ubuntu-14.04"
+    arteria.vm.hostname = "arteria-dev"
+
+    # Box Specifications
+    arteria.vm.provider :virtualbox do |vb|
+      vb.memory = 1028
+      vb.cpus = 2
+    end
+
+    # NFS-synced directory for pack development
+    # Change "/path/to/directory/on/host" to point to existing directory on your laptop/host and uncomment:
+    config.vm.synced_folder "~/workspace/arteria/arteria-packs", "/arteria-packs", :nfs => true, :mount_options => ['nfsvers=3']
```

Comment:
And if there is a requirement to have it NFS mounted in exactly this way then some further instructions are probably needed, because I'm getting mount problems at the moment when running `vagrant up`: 

```
It appears your machine doesn't support NFS, or there is not an
adapter to enable NFS on this machine for Vagrant. Please verify
that `nfsd` is installed on your machine, and try again. If you're
on Windows, NFS isn't supported. If the problem persists, please
contact Vagrant support.
```

Will look into it a bit more when I've gone through the rest.